### PR TITLE
Add sent by and sent at for altinn AGA and FTR invoices

### LINKED ISSUE #218: Save in Lodo when we sent AGA and FRT to fakturabank
Save these values with user id and timestamp.

The send button should work even if you already sent in once. Overwrite the the user id and timestamp

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #218

### CODE REVIEW

**Comment by mladjo2505** [comment]
Success notice not shown as it was on master.

File: modules/fakturabank/model/fakturabank.class.php [hunk 1]
```diff
@@ -2217,6 +2217,23 @@ public function save_invoice_export_data($data) {
         $_SESSION['oauth_invoice_error'] = "Error: " . $data['result'];
         if ($data['code'] == 403) $_SESSION['oauth_invoice_error'] = "Error: Utilstrekkelige rettigheter i fakturabank!";
       }
+      elseif (isset($_SESSION['altinn_invoice_sending']) && $_SESSION['altinn_invoice_sending']) {
```
Comment (mladjo2505) [inline modules/fakturabank/model/fakturabank.class.php]: Move this elseif as an individual if inside the else block

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Comment by mladjo2505** [comment]
Rename `db/changes/139_add_sent_by_and_sent_at_to_altinnreport4_table.sql` to `db/changes/140_add_sent_by_and_sent_at_to_altinnreport4_table.sql` before merge.

**Pull request merged**